README.md

```diff
@@ -193,20 +193,21 @@ under "Command Line Tools".
 By default, Glow will use a system provided LLVM.  Note that Glow requires LLVM
 5.0 or later. If you have LLVM installed in a non-default location (for
 example, if you installed it using Homebrew on macOS), you need to tell CMake
-where to find llvm using `-DCMAKE_PREFIX_PATH`. For example, if LLVM were
+where to find llvm using `-DLLVM_DIR`. For example, if LLVM were
 installed in `/usr/local/opt`:
 
   ```bash
   cmake -G Ninja ../glow \
       -DCMAKE_BUILD_TYPE=Debug \
-      -DCMAKE_PREFIX_PATH=/usr/local/opt/llvm@7
+      -DLLVM_DIR=/usr/local/opt/llvm@7/lib/cmake/llvm
   ```
 
 If LLVM is not available on your system you'll need to build it manually.  Run
 the script '`/utils/build_llvm.sh` to clone, build and install LLVM in a local
-directory. You will need to configure Glow with the flag `-DCMAKE_PREFIX_PATH`
-to tell the build system where to find LLVM (e.g. the location of
-`llvm_install/` if using `build_llvm.sh`).
+directory. You will need to configure Glow with the flag `-DLLVM_DIR` to tell
+the build system where to find LLVM given the local directory you installed it
+in (e.g. `-DCMAKE_PREFIX_PATH=/path/to/llvm_install/lib/cmake/llvm` if using
```

Comment:
Shouldn't it be `-DLLVM_DIR=/path/to/llvm_install/lib/cmake/llvm`?

Reply:
Yep!